thrifty/cost_variance.sp

```diff
@@ -39,3 +40,12 @@ control "full_month_cost_changes" {
   })
 }
 
+control "full_month_forecast_cost_changes" {
```

Comment:
I forgot for the service control we had a variable, would it make sense to add a new variable for this control as well?